### PR TITLE
PAYARA-3682 Fix for error in gf-client manifest from improper $ replacement

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

File: appserver/appclient/client/acc-standalone/pom.xml [hunk 1]
```diff
@@ -145,7 +146,7 @@ this or the downstream JARs and so attempts to find things fail.
                                 <packageName>org.glassfish.appclient.client.acc</packageName>
                                 <addClasspath>true</addClasspath>
                                 <classpathLayoutType>custom</classpathLayoutType>
-                                <customClasspathLayout>../modules/$${artifact.artifactId}$${dashClassifier?}.$${artifact.extension}</customClasspathLayout>
+                                <customClasspathLayout>../modules/${artifact.artifactId}$${dashClassifier?}.$${artifact.extension}</customClasspathLayout>
```
Comment (Pandrex247) [inline appserver/appclient/client/acc-standalone/pom.xml]: Is the double $$ on the `artifact.extension` ok then? Or did you just miss it?

**Comment by Cousjava** [comment]
Jenkins test please

**Comment by Cousjava** [comment]
something else is going on here...

**Pull request closed without merging**